components/ui/use-toast.ts

```
@@ -0,0 +1,192 @@
+
+const TOAST_LIMIT = 1
+const TOAST_REMOVE_DELAY = 1000000
+
+type ToasterToast = ToastProps & {
+  id: string
+  title?: React.ReactNode
+  description?: React.ReactNode
+  action?: ToastActionElement
+}
+
+const actionTypes = {
```

ESLint

[error] 'actionTypes' is assigned a value but only used as a type.

(@typescript-eslint/no-unused-vars)